tests/EventTest.php

```diff
@@ -55,44 +37,26 @@ final class EventTest extends TestCase
 
     protected function setUp(): void
     {
-        $this->uuidGeneratorInvokationCount = 0;
-        $this->originalUuidFactory = new UuidFactory();
         $this->client = ClientBuilder::create()->getClient();
         $this->options = $this->client->getOptions();
```

Comment:
Change not strictly related to this PR, but it doesn't seems to me we're using neither of the two properties above in the class so we can drop them with the `setUp` method

Reply:
Good catch. Fixed